test/unit/participant_test.rb

```diff
@@ -57,12 +65,93 @@ class ParticipantTest < ActiveSupport::TestCase
     end
 
     context "Testing participants" do
+
+
+      should "show that dependency on checkout works" do
+        assert_equal 1, Checkout.all.size        
+        @participant.destroy
+        assert_equal 0, Checkout.all.size        
+      end
+
+      should "show that dependency on membership works" do
+        assert_equal 1, Membership.all.size        
+        @participant.destroy
+        assert_equal 0, Membership.all.size        
+      end
+
+      should "show that dependency on shift_participant works" do
+        assert_equal 1, ShiftParticipant.all.size        
+        @participant.destroy
+        assert_equal 0, ShiftParticipant.all.size        
+      end
+
+      should "show that dependency on organization_status works" do
+        assert_equal 1, OrganizationStatus.all.size        
+        @participant.destroy
+        assert_equal 0, OrganizationStatus.all.size        
+      end
+
+      should "show that dependency on user works" do
+        assert_equal 1, User.all.size        
+        @participant.destroy
+        assert_equal 0, User.all.size        
+      end
+
+      should "show that search scope works properly" do
+        assert_equal [@participant], Participant.search("agoradia")
+        assert_equal [], Participant.search("rkelly")
+      end
+
+      should "show that scc scope works properly" do
+        assert_equal [@participant], Participant.scc
+      end
+
       should "correctly format a participant's phone number" do
         assert_equal "(123) 456-7890", @participant.formatted_phone_number
-
         assert_equal "N/A", @temp_participant.formatted_phone_number
       end
+
+      should "show that is_booth_chair method works correctly" do
+        assert_equal true, @participant.is_booth_chair?
+      end
+
+      should "show that is_scc method works correctly" do
+        assert_equal true, @participant.is_scc?
+      end
+
+      should "show that card_number and card_number= methods works correctly" do
+        @participant.card_number=("818603330")
```

Comment:
I think we can forgo testing this method since testing it would mean having you card number be publicly accessible. We should just remove this test for now.